### PR TITLE
3.4 dependency check

### PR DESCRIPTION
**Pull request by praveenag**
* Create owasp security report for dependencies.
* Upgrade dependent libraries to latest versions.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -864,17 +884,17 @@
...
         <groupId>com.github.ben-manes.caffeine</groupId>
         <artifactId>caffeine</artifactId>
-        <version>2.3.3</version>
+        <version>2.6.1</version>
       </dependency>
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-nop</artifactId>
-        <version>1.7.22</version>
+        <version>1.8.0-beta0</version>
```
Comment (MishaDemianenko) [inline pom.xml]: beta0? I do not think we should have any beta dependencies in the product

File: package-depencies.txt [hunk 2]
```diff
@@ -0,0 +1,5217 @@
+[INFO] Scanning for projects...
```
Comment (MishaDemianenko) [inline package-depencies.txt]: also I do not understand why do we need this file in a repo?

**Review by MishaDemianenko — CHANGES_REQUESTED** [review]

File: package-depencies.txt (hunk 2 above)
Comment (praveenag) [reply package-depencies.txt]: Oh that was added by mistake. Let me remove it.

File: pom.xml (hunk 1 above)
Comment (praveenag) [reply pom.xml]: I agree. That missed my initial sweep. I will update it to the latest patch`1.7.25`.

**Comment by praveenag** [comment]
@MishaDemianenko How about now?

File: pom.xml [hunk 3]
```diff
@@ -1025,17 +1045,17 @@
...
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.5</version>
+        <version>3.7</version>
       </dependency>
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-text</artifactId>
-        <version>1.1</version>
+        <version>1.2</version>
```
Comment (MishaDemianenko) [inline pom.xml]: i remember this one should be the same as in caps project and 1.1 was chosen exactly for that reason not so long time ago

File: community/bolt/LICENSES.txt [hunk 4]
```diff
@@ -15,6 +15,7 @@ Apache Software License, Version 2.0
   Lucene Memory
   Lucene QueryParsers
   Netty/All-in-One
+  Objenesis
```
Comment (MishaDemianenko) [inline community/bolt/LICENSES.txt:20]: do you know where Objenesis comes from now? I remember we killed it some time ago. Do we really need it back or we can exclude it?

**Review by MishaDemianenko — CHANGES_REQUESTED** [review]
couple of comments

**Comment by praveenag** [comment]
Merged after blessings from @MishaDemianenko 🙏

**Pull request merged**